<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>605</width>
    <height>533</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <item>
    <widget class="QTabWidget" name="tabWidget">
     <property name="currentIndex">
      <number>1</number>
     </property>
     <widget class="QWidget" name="widget">
      <attribute name="title">
       <string>学生信息查询</string>
      </attribute>
      <layout class="QHBoxLayout" name="horizontalLayout" stretch="1,2">
       <item>
        <layout class="QVBoxLayout" name="verticalLayout" stretch="0,0,0,0">
         <property name="spacing">
          <number>0</number>
         </property>
         <property name="topMargin">
          <number>100</number>
         </property>
         <property name="bottomMargin">
          <number>100</number>
         </property>
         <item>
          <widget class="QLineEdit" name="num_edit">
           <property name="placeholderText">
            <string>学号：</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="name_edit">
           <property name="placeholderText">
            <string>姓名：</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QComboBox" name="major_combobox">
           <property name="placeholderText">
            <string>专业：</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="search_button">
           <property name="text">
            <string>查询</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <widget class="QTableView" name="tableView"/>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tab">
      <attribute name="title">
       <string>查询学生课程信息</string>
      </attribute>
      <layout class="QHBoxLayout" name="horizontalLayout_2" stretch="1,2">
       <item>
        <layout class="QVBoxLayout" name="verticalLayout_3">
         <item>
          <widget class="QLineEdit" name="num_edit2">
           <property name="placeholderText">
            <string>学号：</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QTextEdit" name="total_score_edit"/>
         </item>
         <item>
          <widget class="QPushButton" name="search_button2">
           <property name="text">
            <string>查询</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <widget class="QTableView" name="tableView_2"/>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tab_2">
      <attribute name="title">
       <string>查询学业预警信息</string>
      </attribute>
      <layout class="QHBoxLayout" name="horizontalLayout_3" stretch="1,2">
       <item>
        <layout class="QVBoxLayout" name="verticalLayout_4">
         <item>
          <widget class="QPushButton" name="search_button3">
           <property name="text">
            <string>查询</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <widget class="QTableView" name="tableView_3"/>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
